### DISCUSSION TITLE
Setting standby frequency on RMP L VHF1

### DISCUSSION THREAD

**Discussion by Muyz** [flybywiresim/aircraft#7265] [Help] [answered]
Am implementing software to control the FBW from HW. So far, I have FCU, and EFIS working just fine. However, with the RMP I am facing an issue that LVAR `A32NX_RMP_L_VHF1_STANDBY_FREQUENCY` is not present. Other LVARS are present, and R/W, such as `A32NX_RMP_L_VHF2_STANDBY_FREQUENCY` and `A32NX_RMP_L_VHF2_STANDBY_FREQUENCY`. Is this intentional or is it a bug?

According to the API documentation, one should be able to set the standby frequency by writing SimConnect variable named `COM STANDBY FREQUENCY:1`. This variable, however, seems to be read-only.

**Comment by tracernz** [comment, marked as answer]
The sim COM frequency variables are used where available. The reason there is an Lvar for VHF2 standby is because the sim does not support that. You probably need to use a K event to set the sim radio frequencies. Refer to https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/Aircraft_Radio_Navigation_Events.htm

**Reply by Muyz** [reply]
Great! Why didn't I think of that ;-) Thanks!
